server/index.js

```diff
@@ -1,13 +1,50 @@
 const path = require('path');
-
 const express = require('express');
+const helpers = require('./helpers');
 
 const app = express();
 const port = 3000;
 
 app.use(express.static(path.join(__dirname, '..', 'public')));
 app.use(express.json());
 
+//Products GET /products Retrieves the list of products
+app.get('/products', (req,res) => {
+  helpers.getProductsList()
+    .then((response) => {
+      res.send(response.data)
+    })
+    .catch((error) => {
+      console.log(error.data);
```

Comment:
We might want `error` here instead of `error.data`, but I'm not 100% certain. I think we should put a ticket in the team backlog to write some simple Jest tests that check the server responses in both the success and error cases for these endpoints. That should clear up exactly what the error looks like, and I suppose we should be testing those sorts of things eventually, anyway.